projects/commudle-admin/src/app/components/sidebar-menu/sidebar-menu.component.html

```diff
@@ -59,17 +46,17 @@
 
           <a [routerLink]="['/admin', 'communities', community.slug]" (click)="closeSidebar()">
             <nb-user size="tiny"
-                name="{{community.name}}"
-                picture="{{ community.logo_path }}">
+                     name="{{community.name}}"
+                     picture="{{ community.logo_path }}">
```

Comment:
At other places the indentation was reduced, but here increased, any reason? :P

Reply:
It is done so that the attributes are in the same vertical indentation. If it isn't needed, I can change it back

![image](https://user-images.githubusercontent.com/60032753/107866740-cb03af00-6e99-11eb-9b42-5b6a3946d0fe.png)

Reply:
Oh, cool, got it. I thought it was something from Linter